test/TensorFlowRuntime/tensor.swift

```diff
@@ -778,6 +778,25 @@ func testShapeGetter4() {
 }
 TensorTests.testAllBackends("ShapeGetter4", testShapeGetter4)
 
+@inline(never)
+func testReplacing1() {
+  let original = Tensor<Float>([[1.0, 0.0], [3.0, 0.0], [2.0, 3.0], [1.0, 0.0]])
+  let modified = original.replacing(with: Tensor<Float>(-1).broadcast(like: original), where: original.<3)
+  let expected = Tensor<Float>([[-1.0, -1.0], [3.0, -1.0], [-1.0, 3.0], [-1.0, -1.0]])
+  expectEqual(expected, modified)
+}
+TensorTests.testAllBackends("Replacing1", testReplacing1)
+
+@inline(never)
```

Comment:
No need to mark these as `@inline(never)` (some other tests were written this way for historical technical constraints). And could you use the trailing closure syntax instead? 

```swift
TensorTests.testAllBackends("replacing(with:where:) with broadcasting") {
   let original = Tensor<Float>([[1.0, 0.0], [3.0, 0.0], [2.0, 3.0], [1.0, 0.0]])
   let modified = original.replacing(with: Tensor<Float>(-1), where: original.<3)
   let expected = Tensor<Float>([[-1.0, -1.0], [3.0, -1.0], [-1.0, 3.0], [-1.0, -1.0]])
   expectEqual(expected, modified)
}
```

Reply:
Great, this is way better.